### PR TITLE
Added cta component and examples

### PR DESCRIPTION
**Pull request by boxadesign**
CTA component.

Button as link component.

### CODE REVIEW

File: src/components/call-to-action/_macro.njk [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% macro onsCallToAction(params) %}
+    {% from "components/button/_macro.njk" import onsButton %}
+    <div class="call-to-action">
+        <div class="container">
+            <div class="grid grid--flex grid--vertical-center grid--no-wrap@s">
+                <div class="grid__col col-auto u-flex-shrink@s">
+                    <p class="call-to-action__desc u-fs-r u-mb-no@s"><strong>{{ params.headingText }}</strong> {{ params.paragraphText }}</p>
```
Comment (bameyrick) [inline src/components/call-to-action/_macro.njk]: After our visit to DAC yesterday I have a feeling`<strong>` tags cause screen readers to pause. I'll check this on JAWS, NVDA, and VoiceOver today

File: src/views/layouts/_master.njk [hunk 2]
```diff
@@ -6,8 +6,14 @@
     {% set page = {} %}
 {% endif %}
 
+
 {% if example %}
     {% set bodyClasses = bodyClasses + " patternlib-page--example" %}
+    {% if page.theme %}
```
Comment (bameyrick) [inline src/views/layouts/_master.njk]: What's this for?

Comment (boxadesign) [reply src/views/layouts/_master.njk]: So we can add a theme to an individual example. Look at the deployment.

File: src/components/call-to-action/_macro.njk (hunk 1 above)
Comment (jrbarnes9) [reply src/components/call-to-action/_macro.njk]: Yes we saw that too @bameyrick. Maybe test if it does the same with `<em>`

File: src/components/call-to-action/_macro.njk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+                </div>
+                <div class="grid__col col-auto u-flex-no-grow">
+                    {{ onsButton({
+                        "text": "Start survey",
```
Comment (jrbarnes9) [inline src/components/call-to-action/_macro.njk]: Doesn't this need to be a parameter?

Comment (boxadesign) [reply src/components/call-to-action/_macro.njk]: Yes

File: src/components/call-to-action/_macro.njk (hunk 1 above)
Comment (boxadesign) [reply src/components/call-to-action/_macro.njk]: JAWS and voiceover reads out tag by tag. The string in the strong tag could be a heading which would be announced in the same way. Perhaps it should be a heading so screen readers would read it out if all headings are read out?

Comment (jrbarnes9) [reply src/components/call-to-action/_macro.njk]: Not sure if matters if screen reader pauses in this instance.

Comment (bameyrick) [reply src/components/call-to-action/_macro.njk]: If it is a heading and a description, wouldn't it make more sense for this to be a heading tag followed by a paragraph tag then?

Comment (boxadesign) [reply src/components/call-to-action/_macro.njk]: Yes that's what I'm saying. I'll update to be a `h2` and `p`

File: src/views/layouts/_master.njk (hunk 2 above)
Comment (bameyrick) [reply src/views/layouts/_master.njk]: Ah cool I see what you've done here, I was an idiot and thought this was the main page-template.njk

**Comment by bameyrick** [comment]
[ Get Started button on homepage doesn't link to anywhere](https://trello.com/c/HegoTuy1/267-get-started-button-on-homepage-doesnt-link-to-anywhere)

**Pull request merged**